File: SDKs/beezup.com/2.0/python/src/sdk/models/shared/companyinfo.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils


@dataclass_json
@dataclasses.dataclass
class CompanyInfo:
    address: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('address') }})
    city: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('city') }})
    company: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('company') }})
    country_iso_code_alpha3: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('countryIsoCodeAlpha3') }})
    postal_code: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('postalCode') }})
    accounting_emails: Optional[list[str]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('accountingEmails') }})
    vat_number: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('vatNumber') }})
    

File: SDKs/beezup.com/2.0/typescript/src/sdk/models/shared/companyinfo.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class CompanyInfo extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=accountingEmails" })
  accountingEmails?: string[];

  @SpeakeasyMetadata({ data: "json, name=address" })
  address: string;

  @SpeakeasyMetadata({ data: "json, name=city" })
  city: string;

  @SpeakeasyMetadata({ data: "json, name=company" })
  company: string;

  @SpeakeasyMetadata({ data: "json, name=countryIsoCodeAlpha3" })
  countryIsoCodeAlpha3: string;

  @SpeakeasyMetadata({ data: "json, name=postalCode" })
  postalCode: string;

  @SpeakeasyMetadata({ data: "json, name=vatNumber" })
  vatNumber?: string;
}
